tools/IBM/writeIBMFiles.m

```diff
@@ -896,10 +912,26 @@
 % zlim([0 zsize])
 
 % Uncomment to view solid points
-%scatter3(xgrid_u(solid_ijk_u(:,1)),ygrid_u(solid_ijk_u(:,2)),zgrid_u(solid_ijk_u(:,3)),10,[0,0,1],'filled')
-%scatter3(xgrid_v(solid_ijk_v(:,1)),ygrid_v(solid_ijk_v(:,2)),zgrid_v(solid_ijk_v(:,3)),10,[0,0,1],'filled')
-%scatter3(xgrid_w(solid_ijk_w(:,1)),ygrid_w(solid_ijk_w(:,2)),zgrid_w(solid_ijk_w(:,3)),10,[0,0,1],'filled')
-%scatter3(xgrid_c(solid_ijk_c(:,1)),ygrid_c(solid_ijk_c(:,2)),zgrid_c(solid_ijk_c(:,3)),10,[0,0,1],'filled')
+%if isempty(solid_IB_ijk_u)
```

Comment:
I would say viewing the solid_IB points is not necessary - can we keep it how it was before (just plot solid points)?

Reply:
Yes I agree that it is not necessary. But, would prefer keeping them as those plots are mainly for checking bugs and to double sure solid-fluid point classification are correct for all grids.

Reply:
Ok, that's no problem - will merge.